### DISCUSSION TITLE
Vector database support? (pgvector)

### DISCUSSION THREAD

**Discussion by tamis-laan** [cockroachdb/cockroach#124794] [Ideas]
I'm wondering if cockroachdb supports vector indexing such as the pgvector plugin for postgresql? 

https://github.com/pgvector/pgvector 

This would be an amazing addition to cockroachdb making it an incredible powerful distributed vector database very relevant to AI.

**Comment by yuzefovich** [comment]
No, we don't support it (yet). The tracking issue is https://github.com/cockroachdb/cockroach/issues/121432, and we do have a prototype in progress https://github.com/cockroachdb/cockroach/pull/124292.
